src/modules/leaderboardApi.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable */
+import { setLocalStorageID, setLocalStorage } from './localStorageFunctions.js';
+
+/* --------===========  Generate Games ID ===========-------- */
+export const createGame = async (loadPage) => {
+  let id;
+  await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(
+        {
+          name: 'Nunito Game',
+        },
+      ),
+    },
+  )
+    .then((data) => data.json()).then((data) => data.result.split(' ')[3])
+    .then((data) => {
+      id = data;
+      setLocalStorageID(id);
+      loadPage();
+    });
+};
+
+/* --------===========  Post New Score ===========-------- */
+export const postScore = async (urlID, user, score) => {
+  await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${urlID}/scores`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(
+      {
+        user,
+        score,
+      },
+    ),
+  }).then((data) => data.json());
+};
+
+/* --------===========  Get Scores ===========-------- */
+
+export const getScores = async (urlID) => {
+  await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${urlID}/scores`)
+    .then((data) => data.json())
+    .then((data) => {
+      data = data.result;
+    })
+    .then((data) => {
+      setLocalStorage(data);
+    });
+};
```

Comment:
- [ ] Your app does not work as expected. Your code shows an error when I try to refresh the leaderboard list by clicking on the refresh button. The submit button should hit the POST API endpoint and post user entries. When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change. Please work on that.
<img width="373" alt="image" src="https://user-images.githubusercontent.com/77190203/180626438-ac2ebecf-b1f2-4c34-9914-2b14d7e814af.png">

Reply:
I didn't get that error in my machine nor in the live version. But the live version was a little outdated, so maybe that happened in some commit. So I changed the code and also gonna update the live version. 

![image](https://user-images.githubusercontent.com/47539402/180627244-97efec63-17ea-4c26-9614-488f281730b1.png)
![image](https://user-images.githubusercontent.com/47539402/180627249-5bcdce89-2536-4f25-9ca7-f81b7f65b5c1.png)

Reply:
Hi @Hannziegel, I tested your project locally and it was showing that error after clicking on the Refresh button. Your app did not work as expected.